File: youlexuan-manage-web/src/main/webapp/js/controller/loginController.js
app.controller("loginController",function($scope,$controller,loginService){
	
	$controller('baseController',{$scope:$scope});//继承
	
	$scope.getLoginName = function(){
		loginService.getLoginName().success(function(response){
			$scope.loginName=response.loginName;
		});
	}
})

File: youlexuan-manage-web/src/main/java/com/offcn/controller/LoginController.java
package com.offcn.controller;

import java.util.HashMap;
import java.util.Map;

import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/login")
public class LoginController {
	
	
	@RequestMapping("/name")
	public Map getLoginName() {
		String userName = SecurityContextHolder.getContext().getAuthentication().getName();
		Map map = new HashMap();
		map.put("loginName", userName);
		return map;
	}
}
